pt-BR/mail/chrome/messenger/imapMsgs.properties

```diff
@@ -133,23 +133,23 @@
 ## @name IMAP_STATUS_GETTING_NAMESPACE
 ## @loc None
 # LOCALIZATION NOTE (Error 5020): Do not translate the word "IMAP" below.
-5020=Checking IMAP Namespace...
+5020=Verificando o namespace IMAP...
 
 ## @name IMAP_UPGRADE_NO_PERSONAL_NAMESPACE
 ## @loc None
-5021="The mail server indicates that you may not have any personal mail folders. Please verify your subscriptions.
+5021="O servidor de e-mails indica que você talvez não tenha pastas de e-mail pessoais. Por favor, verifique suas inscrições.
 
 ## @name IMAP_UPGRADE_PROMPT_USER_2
 ## @loc None
-5024=Any folders which are left unsubscribed will not appear in your folder lists, but can be subscribed to later by choosing File / Subscribe.
+5024=Pastas deixadas como não inscritas não aparecerão nas suas listas de pastas, mas pode-se inscrevê-las depois usando a opção Arquivo / Inscrever.
```

Comment:
CORRECTIONS NEEDED it should be 5024=Pastas deixadas como não inscritas não aparecerão nas suas listas de pastas, mas podem ser subscritas depois usando a opção Arquivo / Inscrever.